web-api/src/persistence/postgres/caseDeadlines/getCaseDeadlinesByDateRange.ts

```diff
@@ -67,3 +69,9 @@ export const getCaseDeadlinesByDateRange = async ({
     totalCount,
   };
 };
+
+const applyJudgeFilter = (query, judgeId) => {
+  return !!judgeId || judgeId === null
+    ? query.where('associatedJudgeId', '=', judgeId)
```

Comment:
I am worried about this. SQL wants `IS NULL` and not `= NULL`. I don't know if Kysely accounts for this?

For example, here are the results I get if I run = NULL:

<img width="935" alt="Screenshot 2025-03-28 at 9 25 48 AM" src="https://github.com/user-attachments/assets/1a24822f-2c0e-4d2c-8df1-983387025f2d" />

And is NULL:

<img width="954" alt="Screenshot 2025-03-28 at 9 25 58 AM" src="https://github.com/user-attachments/assets/5c0fe382-0c2a-4fd0-b97d-111fdab17e69" />


Even if it works with kysely, it is probably safer to do = if judgeId !== null and is if judgeId is null